File: Develop/src/Employee.java
import java.util.Scanner;

public class Employee {

    private String name;
    private String lastName;
    private BankAccount account;
    public Employee(String name, String lastName) {
        this.name = name;
        this.lastName = lastName;
    }

    public Employee(String name, String lastName, long accountNumber,char type) {
        if(type!='A'&&type!='B'&&type!='C'){
            System.out.println("Tipo de cuenta invalido");
            System.out.println("Recuerda que solo hay tipo 'A','B'y'C'");
            System.out.println("Ingrese de nuevo el tipo de cuenta:");
            Scanner leer=new Scanner(System.in);
            type=leer.nextLine().toUpperCase().charAt(0);
            this.name = name;
            this.lastName = lastName;
            this.account = new BankAccount(accountNumber,type);

        }else{
            this.name = name;
            this.lastName = lastName;
            this.account = new BankAccount(accountNumber,type);}


    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public BankAccount getAccount() {
        return account;
    }

    public void mostrarInfoAll(){

    }
}

File: Employee.py
import random


class Employee:
    def __init__(self, name, last_name, account_number=None, type=None):
        if type not in ['A', 'B', 'C']:
            print("Tipo de cuenta invalido")
            print("Recuerda que solo hay tipo 'A', 'B' y 'C'")
            print("Ingrese de nuevo el tipo de cuenta:")
            type = input().upper()

        self.name = name
        self.last_name = last_name
        if account_number is not None and type is not None:
            self.account = BankAccount(account_number, type)
        else:
            self.account = None

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_last_name(self):
        return self.last_name

    def set_last_name(self, last_name):
        self.last_name = last_name

    def get_account(self):
        return self.account

    def mostrar_info_all(self):
        pass


class BankAccount:
    def __init__(self, account_number, account_type):
        self.account_number = account_number
        self.account_type = account_type

    def agregar_dinero(self, amount):
        pass

    def quitar_dinero(self, amount):
        pass


class Bank:
    def __init__(self):
        pass

    def validacion_num(self, account_number):
        pass

    def clientes(self, employee):
        pass

    def info(self, espacio):
        pass

    def log_in(self, name, last_name, account_number):
        pass

    def imprimir_clientes(self):
        pass

    def buscar_user(self, name, last_name):
        pass


def main():
    band = True
    clientes = Bank()
    persona = None
    amount = 0
    validacion = ""
    espacio = 0

